.github/workflows/pr.yaml

```
@@ -141,16 +143,12 @@ jobs:
         sccache -s;
 
   telemetry-summarize:
-    runs-on: ubuntu-latest
+    # It is essential that this job run on a self-hosted runner. Otherwise, the job
+    # won't be able to communicate with FluentBit, and you won't be recording anything.
```

Can we simplify this? RMM developers won't know what FluentBit is (I don't know what it is either) but it seems like an implementation detail that doesn't need to be mentioned in every repo.
```suggestion
    # This job must use a self-hosted runner to record telemetry traces.
```